<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Split</class>
 <widget class="QWidget" name="Split">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Split</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="12">
    <widget class="QSpinBox" name="sb_seconds"/>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="QPushButton" name="pb_define_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="7" column="8" colspan="5">
    <widget class="QPushButton" name="pb_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="13">
    <widget class="QTextEdit" name="txte_affichage">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="12" colspan="2">
    <widget class="QToolButton" name="btn_open">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="5">
    <widget class="QTimeEdit" name="te_sound">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="pb_play_subsound">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QProgressBar" name="pb_level">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <widget class="QTimeEdit" name="te_start_subsound">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MinuteSection</enum>
     </property>
     <property name="displayFormat">
      <string>mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pb_munus">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="12">
    <widget class="QLabel" name="lbl_file">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="11">
    <widget class="QPushButton" name="pb_plus">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8" colspan="5">
    <widget class="QTimeEdit" name="te_end_subsound">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8" colspan="5">
    <widget class="QPushButton" name="pb_define_end">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QToolButton" name="tb_save_folder">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="4">
    <widget class="QPushButton" name="pb_play_sound">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLabel" name="lbl_save">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QPushButton" name="pb_pause_sound">
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
